src/core/worker.lua

```diff
@@ -28,16 +28,23 @@ function start (name, luacode)
    if pid == 0 then
```

Comment:
Can you please try adding
```
      shm.mkdir(shm.resolve("group/dma"))
```
before line 27 and keeping the shm.alias line uncommented? It seems like a simple fix that would prevent diffs with upstream and the need to comment.